ns1/README.md

```diff
@@ -1,84 +1,62 @@
+## Installation
 
-1. [Download and launch the Datadog Agent][4].
-2. Run the following command to install the integrations wheel with the Agent:
+Refer to Datadog’s [community integrations installation](https://docs.datadoghq.com/agent/guide/community-integrations-installation-with-docker-agent/?tab=agentabovev68) page for the specific steps to install the NS1 integration as an add-on, since it is not packaged and built into the Datadog Agent. 
 
-   ```shell
-   datadog-agent integration install -t datadog-ns1==0.0.2
-   ```
-   
-3. Configure your integration like [any other packaged integration][5].
+Note that the minimum Agent version required for supporting integrations is 7.21.0. NS1 recommends using the latest version of the Datadog Agent.
 
+When running the installation process, make sure to do so using the following variables specific to the NS1 integration:
 
 
-### Configuration
+* <INTEGRATION_NAME>:  NS1
+* <INTEGRATION_VERSION>:  0.0.3
 
-1. Edit the `ns1.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your ns1 performance data. See the [sample ns1.d/conf.yaml](datadog_checks/ns1/data/conf.yaml.example) for all available configuration options.
 
-2. [Restart the Agent][7].
 
-For detailed explanation of all collected performance data see the [NS1 + Datadog Integration][12] article in NS1 help center.
+## Configuration
 
-### Validation
-
-[Run the Agent's status subcommand][8] and look for `ns1` under the Checks section.
+To configure and activate the NS1 integration, follow the steps [here](https://docs.datadoghq.com/getting_started/integrations/#configuring-agent-integrations). 
 
-## Data Collected
+See the [sample ns1.d/conf.yaml](https://github.com/DataDog/integrations-extras/blob/master/ns1/datadog_checks/ns1/data/conf.yaml.example) for all available configuration options.
 
-### Metrics
 
-See [metadata.csv](metadata.csv) for a list of metrics provided by this check.
+### Validation
 
-### Service Checks
+To validate your Agent and integration configuration, [run the Agent's status subcommand](https://docs.datadoghq.com/agent/guide/agent-commands/#agent-status-and-information) and look for ns1 under the Checks section.
 
-ns1 does not include any service checks.
 
-### Events
+## Data Collected
 
-ns1 does not include any events.
+## Metrics
 
-### Development
+See [metadata.csv](https://github.com/DataDog/integrations-extras/blob/master/ns1/metadata.csv) for a list of metrics provided by this integration.
 
-See the [Datadog developer documentation][10] for more information about how to test and develop Agent based integrations.
+## Service Checks
 
-To install the ns1 check on your host:
+The NS1 integration sends the following service check after each run:
+ns1.can_connect returns CRITICAL if the Agent fails to receive a 200 from the NS1 API endpoint or returns OK if all endpoints respond properly.
```

Comment:
```suggestion
The NS1 integration sends the following service check after each run:

`ns1.can_connect` returns CRITICAL if the Agent fails to receive a 200 response code from the NS1 API endpoint or returns OK if all endpoints respond properly.
```

needs extra new line

Reply:
```suggestion
See [service_checks.json][8] for a list of service checks provided by this integration.
```
This "magic sentence" pulls in the service check info from the config file and formats it consistently (like the Data Collected magic sentence does for metrics). @ruthnaebeck Can you confirm I've done this correctly?